code/game/objects/items/cards_ids.dm

```diff
@@ -441,14 +441,11 @@
 /obj/item/card/id/add_context(atom/source, list/context, obj/item/held_item, mob/user)
 	. = ..()
 
-	if(held_item != src)
-		return
-
 	context[SCREENTIP_CONTEXT_LMB] = "Show ID"
 	context[SCREENTIP_CONTEXT_RMB] = "Project pay stand"
 	if(isnull(registered_account) || registered_account.replaceable) //Same check we use when we check if we can assign an account
-		context[SCREENTIP_CONTEXT_ALT_LMB] = "Assign account"
-	else
+		context[SCREENTIP_CONTEXT_ALT_RMB] = "Assign account"
+	if(!registered_account.replaceable || registered_account.account_balance > 0)
```

Comment:
This doesn't look safe to me, it'll runtime if `registered_account` is `null`?

Reply:
My understanding is that registered account is currently never null. On blank ID cards, an unassigned account is made, and unassigned accounts are the subtype with replaceable so that they can be replaced by a players real ID number should they lose their original ID.

I could throw in a sanity check though no issue if you'd prefer

Relevant:
```
/obj/item/card/id/Initialize(mapload)
	. = ..()
	var/datum/bank_account/blank_bank_account = new /datum/bank_account("Unassigned", player_account = FALSE)
	registered_account = blank_bank_account
	blank_bank_account.account_job = new /datum/job/unassigned
	registered_account.replaceable = TRUE
```

Reply:
Pain, i'll throw together a 1line fix today

Reply:
I was wondering about how "registered account is never null" but the line right above that I highlighted checked `isnull(registered_account)`